app/_layout.tsx

```
@@ -19,13 +20,25 @@
   });
-  const { token, refreshToken, isInitialized } = useAuth();
 
-  if (!isInitialized || !loaded) {
-    return null;
-  }
+  let token: string | null = null;
+  let refreshToken: string | null = null;
+  let isInitialized = false;
+
+  try {
+    const authData = useAuth();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)